comp/otelcol/otlp/components/processor/infraattributesprocessor/taggerclient_test.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package infraattributesprocessor
+
+import (
+	"github.com/DataDog/datadog-agent/comp/core/tagger/taggerimpl/collectors"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
+)
+
+type testTaggerClient struct {
+	m map[string][]string
```

Comment:
nit: a more descriptive name than `m` would be helpful here

Reply:
great feedback, made a change.